.github/workflows/prerelease_linux.yml

```diff
@@ -0,0 +1,114 @@
+name: Prerelease Linux pipeline
+
+on:
+  release:
+    types:
+      - prereleased
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GO_VERSION: '1.14.4'
+  GPG_MAIL: 'infrastructure-eng@newrelic.com'
+  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+  SNYK_TOKEN: ${{ secrets.COREINT_SNYK_TOKEN }}
+  TAG: ${{ github.event.release.tag_name }}
+  DOCKER_HUB_ID: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+  DOCKER_HUB_PASSWORD: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+
+jobs:
+  unit-test-linux:
+    name: Linux unit tests
```

Comment:
Where are the Windows unit tests being run on pre-release?

Reply:
I've decided to split the windows/linux workflows to be able to restart just one in case is needed
the tests for windows are here:
https://github.com/newrelic/infrastructure-agent/pull/298/files#diff-baffe0a13a846064187256b5ad71d44ca7f65a41dc9d51466b35580e957771f4R19